### PR TITLE
Test coverage increase

### PR DESCRIPTION
**Pull request by GSinseswa721**
**Description**

This PR is all about increasing the coverage of our project. Currently it had a test coverage of 77% , we have to increase it at least up to 90%. We are going to write tests to cover untested and partially tested areas of the code.

**Description of task to be completed**

- [ ] The test coverage increase at least to 90%.
- [x] No functionality is broken by the new test.
- [ ] All test pass successfully.

### CODE REVIEW

**Comment by faid-terence** [comment]
@GSinseswa721 and @Ndevu12 
What's the progress ?

**Comment by Ndevu12** [comment]
> @GSinseswa721 and @Ndevu12 
> What's the progress ?

We have managed to achieve 81% for now

**Review by faid-terence — CHANGES_REQUESTED** [review]
@GSinseswa721 and @Ndevu12 
We need  acoverage of 90+%

File: src/__test__/cart.test.ts [hunk 1]
```diff
@@ -625,4 +625,4 @@ describe('Cart| Order  management for guest/buyer', () => {
       expect(response.body.data.cart).toBeDefined;
     });
   });
-});
+});
```
Comment (aimedivin) [inline src/__test__/cart.test.ts:866]: @Ndevu12 , @GSinseswa721 , please check the 'Order Management Tests' suite starting from line 392 to see if they are written correctly. for ex, test on line 398, 436 (their test names).

**Review by aimedivin — CHANGES_REQUESTED** [review]

**Review by maxCastro1 — APPROVED** [review]

**Pull request closed without merging**